eng/Directory.Build.Common.targets

```diff
@@ -87,7 +87,7 @@
 
     <PackageReference Condition="'$(EnableClientSdkAnalyzers)' == 'true'" Include="Azure.ClientSdk.Analyzers" PrivateAssets="All" />
 
-    <PackageReference Condition="'$(EnableFxCopAnalyzers)' == 'true'" Include="Microsoft.CodeAnalysis.FxCopAnalyzers" PrivateAssets="All" />
+    <PackageReference Condition="'$(EnableNetAnalyzers)' == 'true'" Include="Microsoft.CodeAnalysis.NetAnalyzers" PrivateAssets="All" />
```

Comment:
It's unclear what this could break. You should add to a property group setting EnableNetAnalyzers based on EnableFxCopAnalyzers, which no-ops if neither is set. Add a comment this exists for legacy reasons.

Alternatively, you make this change as is and we see what breaks. @weshaggard @hallipr?

Reply:
I think it would be pragmatic to analyze the differences between the two analyzers and then enable them (once we have an idea of the impact on other SDKs). Looking at the pipelines we only have run core and batch as part of this PR and should run this against other SDKs before merge (it has a potential to break a large set of pipelines and block the code merge \ release).

Reply:
@pallavit Is there a command/script which can run all tests?

Reply:
@pallavit Is there a command or script we can build all projects in this repo?

Reply:
You can build `eng/service.proj` without any service directory to build all of them locally.